lab_1_classify_by_unigrams/main.py

```diff
@@ -4,104 +4,71 @@
 """
 
 
-def tokenize(text: str) -> list[str] | None:
-    """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
+def tokenize(text):
+    text = text.lower()
+    for i in text:
+        if not i.isalnum() and i != ' ':
```

Comment:
if the first condition is true, so is the second, right?

Reply:
By the way, should we keep the digits in the text?

Reply:
I've changed my code and now it chooses only letters